### ISSUE TITLE
Implement `Acts::Logging::OutputPrintPolicy` to funnel output to spdlog.

### ISSUE THREAD

**Issue by veprbl** [eic/EICrecon#971]
The current ACTS logging bypasses the spdlog. That results in an inconsistent formatting.

While implementing this we might as well attempt filtering known uninformative messages like #512 and the now used track propagation.

The #133 suggests there is a similar issue for DD4hep.

**Comment by wdconinc** [comment]
I was looking into this, but I don't know if this is possible. We would need to provide an std::ostream& from spdlog to the Acts lines in e.g. CKFTracking.cc, such as
```cpp
ACTS_LOCAL_LOGGER(Acts::getDefaultLogger("CKFTracking Logger", logLevel, &std::cout));
```
I don't think spdlog provides this, only an ostream sink.

**Comment by wdconinc** [comment]
Fixed by #1012
